server/text_generation_server/layers/compressed_tensors/w8a8_int.py

```diff
@@ -10,7 +11,7 @@
 from text_generation_server.utils.weights import Weight, Weights, WeightsLoader
 
 try:
-    import marlin_kernels
+    marlin_kernels = load_kernel("kernels-community/quantization")
```

Comment:
Should we rename the variable too ?

Reply:
Yeah, I think we should. But probably only for the cases where there is no matching ROCm kernel (e.g. `moe_kernels`)?

Reply:
Fixed.